File: ATV_1_Estruturas_de_selecao/exercicio4.py
meses = [
    "Janeiro",
    "Fevereiro",
    "Março",
    "Abril",
    "Maio",
    "Junho",
    "Julho",
    "Agosto",
    "Setembro",
    "Outubro",
    "Novembro",
    "Dezembro"
]
    
def retornaMesNumero(mes): 
    if mes >= 1 and mes <=12:
        return("O mes {} é {}".format(mes, meses[mes-1]))
    else:
        return("Numero informado não corresponde a nenhum mes!")
    

File: Funcional/Aulas/ATV_1_Estruturas_de_selecao/exercicio4.hs
meses = ["Janeiro", "Fevereiro", "Março", "Abril", "Maio", "Junho", "Julho", "Agosto", "Setembro", "Outubro", "Novembro", "Dezembro"]

retornaMesNumero mes
    | mes >= 1 && mes <= 12 = "O mes " ++ show(mes) ++ " e " ++ show(meses !! (mes - 1))
    |otherwise = "Numero informado não corresponde a nenhum mes!"
